src/Chapter02/Chapter02.csproj

```diff
@@ -6,6 +6,9 @@
     <StartupObject>AddisonWesley.Michaelis.EssentialCSharp.Shared.Program</StartupObject> 
   </PropertyGroup>
   <Import Project="..\Versioning.targets" />
+  <ItemGroup>

```

Comment:
We might want to remove this Listing.. for now Im going to let it compile..but the next commit might see this file being removed
```suggestion
```